.github/workflows/source/test_travis_matrix.sh

```diff
@@ -22,6 +22,8 @@ WARPX_CI_RZ_OR_NOMPI=TRUE      python prepare_file_travis.py
 grep "\[" travis-tests.ini >> travis_matrix_elements.txt
 WARPX_CI_QED=TRUE               python prepare_file_travis.py
 grep "\[" travis-tests.ini >> travis_matrix_elements.txt
+WARPX_CI_PSATD_RZ=TRUE          python prepare_file_travis.py
+grep "\[" travis-tests.ini >> travis_matrix_elements.txt
```

Comment:
Could this be folded into the case `WARPX_CI_PSATD=TRUE`, instead of creating a new element in the Travis matrix?
I think that the reason why we have 6 elements in the Travis matrix is that our free Travis account allows 6 simultaneous run. If we have 7 elements, than one of the runs will not be simultaneous, which may make the tests significantly longer.

Reply:
It was 6 in the past, now it's only 5 free parallel ones :cry: (one more reason to ideally fold it into the other ones if we can).

The best way to do this is to always compile on travis with PSATD enabled and making the field solver a runtime option.

Reply:
Ok, I'll add it to the existing psatd test.

Having the psatd as a runtime option would be nice. It would simplify the Python interface too.